dist/windows/installer.nsh

```diff
@@ -112,6 +112,13 @@ SectionEnd
 ;--------------------------------
 
 Function .onInit
+  ; create a mutex ensures only one installer is running
+  System::Call 'kernel32::CreateMutexA(i 0, i 0, t "qbt_installer") i .r1 ?e'
```

Comment:
Shouldn't you take ownership of the mutex? The example here does: https://nsis.sourceforge.io/Allow_only_one_installer_instance
```suggestion
  System::Call 'kernel32::CreateMutexA(i 0, i 1, t "qbt_installer") i .r1 ?e'
```